includes/admin/payments/class-refund-items-table.php

```diff
@@ -339,6 +351,28 @@ private function get_quantity_column( $item, $column_name, $item_id, $object_typ
 		return ob_get_clean();
 	}
 
+	/**
+	 * Retrieves the number of decimals for a given order.
+	 *
+	 * @param int $order_id
+	 *
+	 * @since 3.0
+	 * @return int|null
+	 */
+	private function get_order_currency_decimals( $order_id ) {
+		static $currency_decimals = null;
+
+		if ( is_null( $currency_decimals ) ) {
+			$order = edd_get_order( $order_id );
+
+			if ( $order ) {
+				$currency_decimals = edd_currency_decimal_filter( 2, $order->currency );
+			}
```

Comment:
@robincornett Maybe we should have an `else` here that sets it to `2`. Unlikely to wind up there, but just in case.

Reply:
Yes, I agree. Committed.